src/components/global/search/_search.scss

```diff
@@ -17,44 +18,53 @@
 
   .search--field {
     flex: 1;
+    position: static !important;
+
+    &:after {
+      content: "";
+      display: inline-block;
+      @include searchbtn;
+      position: absolute;
+      right: 35px;
+      opacity: 0.6;
+    }
   }
 
   .search--input {
     float: right;
     width: 0;
     height: 0;
-    margin: 10px 0 0 0;
-    padding: 0;
+    padding: 10px 50px 10px 20px;
     border: none;
     border-radius: 34px;
     background-color: #ffffff;
-    box-shadow: 0 2px 20px 0 rgba(0, 0, 0, 0.2);
+    box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
     display: none;
     font-family: inherit;
     font-weight: 300;
 
-    &:focus {
-      outline: 2px dotted var(--theme-color-dark);
-      outline-offset: 2px;
-    }
-  }
-
-  .search--input-expanded {
     display: inline;
     width: 100%;
     height: 45px;
-    padding: 10px;
     transition: width 0.25s, height 0.25s, padding 0.25s;
+
+    &:focus {
+      outline: 2px dotted var(--theme-color-dark);
+      outline-offset: 2px;
+    }
   }
 }
 
 // Overrides for the default SiteSearch 360 styles
 #unibox-suggest-box {
-  margin-top: 1px;
-  box-shadow: 0 2px 20px 0 rgba(0, 0, 0, 0.2);
+  margin-top: 46px;
+  box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
   font-family: inherit;
   font-weight: 300;
   border: none;
+  left: auto !important;
+  top: auto !important;
+  right: auto !important;
 
```

Comment:
@ralf57, suggestion regarding the scrolling mentioned by @rudigier:
```
overflow: auto !important;
max-height: calc(100vh - 210px);
````

This works finde for me on both desktop and mobile.